### ISSUE TITLE
Lit Context - no bundle support and no JavaScript documentation

### ISSUE THREAD

**Issue by Neutrino-Sunset** [lit/lit#4778]
### Which package(s) are affected?

Context (@lit/context), Other/unknown (please mention in description)

### Description

The documentation states that lit-all.min.js exports everything in core plus most other modules in lit.

Yet when I create a module

```js
import { LitElement, html, createContext } from 'https://cdn.jsdelivr.net/gh/lit/dist@3/all/lit-all.min.js';

export class SimpleGreeting extends LitElement {
   constructor() {
      super();
   }

   render() {
      return html`<p>Hello, World!</p>`;
   }
}

customElements.define('simple-greeting', SimpleGreeting);
```

I get the error

> The requested module 'https://cdn.jsdelivr.net/gh/lit/dist@3/all/lit-all.min.js' does not provide an export named 'createContext'

The documentation states that I should be able to run Lit from a bundle, but without support for Context that's really not the case.

I was hoping to use Lit to add some reactivity to an ASP Core web app. Integrating a multistage (npm install, create bundle) client build process into Visual Studio in a way that works as a single build step both locally and in a hosted build pipeline is not trivial, and not something anyone wants to have to set up just to experiment with a new library.

Please consist releasing a usefully complete bundle.

Also the Context section of the documentation has no documentation for Javascript at all.

### Reproduction

I think most people should be able to understand the issue from the description.

### Workaround

I have not found a workaround.

### Is this a regression?

No or unsure. This never worked, or I haven't tried before.

### Affected versions

Failing in 3.2.0

### Browser/OS/Node environment

All environments

**Comment by justinfagnani** [comment]
Yes, only the core Lit packages - `lit-html`, `@lit/reactive-element`, `lit-element`, and `lit` are bundled. It's simply not feasible for us to bundle all of the Lit-related packages - there are a significant number of them. If you need to pick and choose packages, you need to use a package manager to install them. Some people use npm locally and check-in the packages so that they don't have to integrate npm with their other tooling.

**Comment by augustjk** [comment]
Other CDNs exist that provide bundles of npm published packages like https://esm.sh/.
You can import with
```js
import {createContext} from 'https://esm.sh/@lit/context';
```

You can even pin to a specific version (range) or modify bundling or transpilation strategies as documented in their site.

Regarding JS docs for context, all the non-decorator examples of using the `ContextProvider` and `ContextConsumer` will work fine with JS. (and decorators are making their way through the standards process and getting implemented so someday will work in JS as well.) Hopefully looking past the typescript annotations aren't too difficult, but indeed making switchable samples that remove those could be an improvement that we just haven't gotten around to.

**Comment by eleses** [comment]
Lit Task is not included in all-min either. That one is more problematic because even old Lab versions of it try to pull the _latest_ version of lit/react. And so Task 3.0.2 that shipped with Lit 2 and back then didn't require ES2021 now does.


The files actually bundled [seem](https://github.com/lit/lit/blob/1889e0eb1cdda28a9e9981ac4657d381e1d1bfbf/packages/lit/rollup.config.js#L97) listed in `index.all`, but I'm not sure that that one is generated because it doesn't seem to be a static file committed to this repo. I'm guessing it's just the list starting at line 35 in the same `rollup.config.js` file. Actually it's referring to [index.all.ts](https://github.com/lit/lit/blob/1889e0eb1cdda28a9e9981ac4657d381e1d1bfbf/packages/lit/src/index.all.ts), rollup drops the extension in its config. But you probably need to update the entry points too in `rollup.config.js`, if you add something there.

Also, that `index.all.ts` is not present in what npm installs from lit, so you need it from here/github, if you want to rebundle with more stuff added. Although `index.all.js` is actually in npm. Perhaps rollup just use that one. 


@Neutrino-Sunset try the 2nd package from https://eleses.github.io/litt/ which include Context now (as well as Task.) I just rebundled it with esbuild and imported both so common stuff like lit react only gets one copy (see the `build.cmd` for details and files it calls). It's transpiled for ES2019 too, so also works on slightly older browsers that don't support `??=` natively. 


Somethig like

```
<script type="module">
	import {LitElement, html, asyncReplace, Task, createContext} from 'https://eleses.github.io/litt/littc-all.js'
	// your code here
</script>
```

worked for me (straight from within CsoundQT for that matter, which bundles a fairly ancient Chrome 83 on Windows.)
